Codeinwp/templates-patterns-collection
# Feat/cleanup import

### Summary
We now store the import changes inside a transient with a 24h lifetime.
After import a new option should be available on subsequent imports to cleanup the previous import:
![image](https://user-images.githubusercontent.com/23024731/118494042-653df200-b72a-11eb-825c-e87e9f915af5.png)

If checked it should cleanup previous import before adding the new import.

### Test instructions
1. Import a new starter site.
2. Toggle the cleanup option on a new import.
3. Check that no settings from the first import are still present. (images, posts, categories, plugins etc.)

Closes Codeinwp/neve/issues/822
